.github/workflows/run-e2ecase.yml

```diff
@@ -40,5 +40,5 @@ jobs:
 
     - name: Run E2ECases
       run: |
-        cd flink-doris-connector && mvn test -Dtest="*E2ECase" -Dimage="apache/doris:doris-all-in-one-2.1.0"
```

Comment:
Referring to the naming of flink, I think E2ECase will be a better name.

Reply:
done